ui/app/styles/components/stat-text.scss

```diff
@@ -104,3 +103,15 @@
     }
   }
 }
+
+.has-gap {
+  gap: $spacing-xxl;
+}
+
+.stat-text-flex-wrapper {
```

Comment:
all this flex business feels untidy, open to clean-up ideas but was also planning to revisit once the feature was fully built rather than get hung up on css styling now

Reply:
If the column and row gap values may become common across the app you could create a reusable class, something like:
```css
.is-flex-wrap-with-gap {
  flex-wrap: wrap;
  column-gap: 70px;
  row-gap: 20px;
}
```
Something like that could be used along with `.is-flex-start`. I think it's fine the way you approached it though, especially if those gap values are specific to the stat text.

Reply:
I think this is good to keep in mind, but right now it seems like styling is unique to this component